client/pages/sections/granted/project-summary.js

```diff
@@ -0,0 +1,125 @@
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import ReactMarkdown from 'react-markdown';
+import Review from '../../../components/review';
+import format from 'date-fns/format';
+
+const ProjectSummary = ({
+  project,
+  values,
+  granted: {
+    licenceHolderLegal,
+    grantedAuthority
+  },
+  establishment: {
+    name,
+    licenceNumber,
+    address,
+    licenceHolder: {
+      firstName,
+      lastName
+    }
+  },
+  fields
+}) => {
+  const ra = (values.conditions || []).find(c => /retrospective-assessment/.test(c.key));
+  return (
+    <Fragment>
+      {
+        ra && (
+          <div className="granted-section">
+            <h3>{ra.title}</h3>
+            <div className="purple-inset">
+              <p>{ra.content}</p>
+            </div>
+          </div>
+        )
+      }
+      <div className="granted-section">
+        <h3>Project licence number</h3>
+        <p>{project.licenceNumber}</p>
+      </div>
+      <div className="granted-section">
+        <h3>Project licence holder</h3>
+        <p>{`${project.licenceHolder.firstName} ${project.licenceHolder.lastName}`}</p>
+        <ReactMarkdown className="legal">{licenceHolderLegal}</ReactMarkdown>
+      </div>
+      <div className="granted-section">
+        <Review
+          {...fields.find(f => f.name === 'duration')}
+          value={values.duration}
+          noComments
+        />
+      </div>
+      <div className="granted-section">
+        <h3>Date granted</h3>
+        <p>{format(project.issueDate, 'DD MMMM YYYY')}</p>
```

Comment:
Might be worth moving these date format strings into their own file somewhere like we do in pages (or even better, somewhere shared across all projects like `asl-constants`)

Reply:
This project doesn't use our artifactory npm registry so has no access to asl-constants else I would have just used that. I do agree these magic strings should be defined somewhere better though - please hold.